core/src/main/java/bio/terra/pearl/core/service/kit/PepperDSMClientProvider.java

```diff
@@ -0,0 +1,35 @@
+package bio.terra.pearl.core.service.kit;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Primary;
+
+@Configuration
+public class PepperDSMClientProvider {
+    private final LivePepperDSMClient livePepperDSMClient;
+    private final StubPepperDSMClient stubPepperDSMClient;
+
+    public PepperDSMClientProvider(LivePepperDSMClient livePepperDSMClient, StubPepperDSMClient stubPepperDSMClient) {
+        this.livePepperDSMClient = livePepperDSMClient;
+        this.stubPepperDSMClient = stubPepperDSMClient;
+    }
+
+    /**
+     * Resolves the PepperDSMClient implementation to use in the deployed application.
+     */
+    @Primary
+    @Bean
+    public PepperDSMClient getPepperDSMClient() {
+        return getStubPepperDSMClient();
```

Comment:
Currently using a stub implementation (to be built out as needed) until we have more working functionality in DSM dev.